package.json

```diff
@@ -4,7 +4,8 @@
   "description": "Help slow the spread of COVID-19 by discouraging people from touching their face while sitting in front of a computer.",
   "main": "index.js",
   "scripts": {
-    "start": "node app.js"
+    "start": "node app.js",
+    "dev": "nodemon app"
```

Comment:
Yes this is fantastic!
Would you mind adding a quick line about it to README similar to the one on start script?

Reply:
Sure, I'll do that now.